README.md

```diff
@@ -4,11 +4,15 @@ The spatial data infrastructure for the [KLIPS project](http://www.klips-projekt
 
 ## Setup
 
-
 ```shell
 # create your personal config
-cp .env.example .env
```

Comment:
i like the old syntax, no need to break things down, but its just personal opinion..